### ISSUE TITLE
Urbit group deleted spontaneously

### ISSUE THREAD

**Issue by somlus-savlev** [urbit/urbit#2816]
Hello I created ship "hiddev-dannut" to host the new Hooniverse group.  A member filsur-sanmut could not see it, even though at some point he had to have had access because he was listed as a colored sigil with access.  I clicked on the sigil and saw the expanded sigil page with his full ship name and the "Remove from group" link in the upper right. upon clicking that link, the entire group disappeared.  It was confirmed to have disappeared from my other ship which was a member.

I recreated the Hooniverse group on that ship and invited a couple of former members. Those members immediately had access to all chat and group links (without re-creating them) and could post and be seen in those chat channels. 

So in summary, a group got deleted from removing a member, and then once the group was re-created the chats and links were all there automatically.  Only the members were gone. I am hoping that re-inviting the members does not lead to more of this disappearing group error.

Here is the error that filsur-sanmut got:

https://pastebin.com/nJ1b10Nr

In case that is helpful.

**Comment by SetKam** [comment]
Huh! That's why I got kicked off the Hooniverse...
I was wondering earlier

**Comment by somlus-savlev** [comment]
Ok we have another potential "pre-delete" situation here.  ~hodler-pinryx cannot get into the New Hooniverse, even though he is seen as being there and having a colored sigil.  

Prior to this, he reported an issue joining, so I went to the New Hooniverse host planet ~hiddev-dannut groups page and saw his sigil _not_ colored up.  Then I removed him from the link next to the list and then re-added him.  He appeared colored up.  I now obviously don't want to remove him from the group or the group might get entirely deleted again like yesterday.

Here is his reported error on his end:
https://hatebin.com/vpttmwmouq

Please let me know when I can go back to safely managing users without a delete issue.  I would actually appreciate an email since I do not check these regularly, which can be sent to somlus at somlus dot com

**Comment by somlus-savlev** [comment]
I should also mention that the effort originally to immediately recreate the group with the same name after auto-deletion yesterday led to some issues with some ships seeing the chat rooms / links / notebooks and others not seeing them. So I went ahead and created an entirely new group with a new base name (i.e. New Hooniverse instead of Hooniverse).  This new issue is occurring in the newly created group, after inviting everyone from scratch.

**Comment by ryjm** [comment]
This just happened to me with ~middev/the-forge. Went to kick and re-invite a member, and when i re-invited the group disappeared.

**Comment by tylershuster** [comment]
I also have this error with ~marpem/permaculture. The group exists and direct links work but it doesn't show up in UIs

**Comment by ryjm** [comment]
Saw the following on ~middev

```
poke failed from %contact-view on wire /
poke-ack
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[145 12].[145 35]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[145 3].[150 75]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[139 3].[150 75]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[137 3].[150 75]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[131 14].[131 59]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[129 3].[133 5]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[128 3].[133 5]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[127 3].[133 5]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[68 9].[68 57]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[63 7].[72 9]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[62 5].[73 17]>
/~rabsef-bicrym/home/0/app/contact-hook/hoon:<[61 5].[73 17]>
/~rabsef-bicrym/home/~2020.4.25..02.43.00..191e/sys/vane/gall:<[1.280 9].[1.280 37]>
; ~palfun-foslup not responding still trying
```

**Comment by ryjm** [comment]
A `:group-store +dbug` on ~middev shows that it's no longer in the app state, but doing the same on my planet shows that it's still there. So at least I can retain the member list.

**Comment by Zaphod101010** [comment]
Running `|merge %home our %base` took all day, but after it completed I'm now able to join the Hooniverse group.
